sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,680 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------
 
-from typing import List, Any, Optional, TYPE_CHECKING
+from typing import Dict, List, Any, Optional, TYPE_CHECKING, Tuple, Union
 
 from azure.core import PipelineClient
+from cryptography.hazmat.primitives import serialization
 from six import python_2_unicode_compatible
 
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
     from typing import Any
 
     from azure.core.credentials import TokenCredential
-    from azure.core.pipeline.transport import HttpRequest, HttpResponse
 
 from ._generated import AzureAttestationRestClient
 from ._generated.models import (
-    AttestationType, 
-    PolicyResult as GeneratedPolicyResult, 
-    PolicyCertificatesResult, 
-    JSONWebKey, 
-    AttestationCertificateManagementBody, 
+    AttestationType,
+    PolicyResult as GeneratedPolicyResult,
+    PolicyCertificatesResult as GeneratedPolicyCertificatesResult,
+    JSONWebKey,
+    AttestationCertificateManagementBody,
     StoredAttestationPolicy as GeneratedStoredAttestationPolicy,
-    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult
+    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult,
 )
 from ._configuration import AttestationClientConfiguration
 from ._models import (
-    AttestationSigner, 
-    AttestationToken, 
-    AttestationResponse, 
-    AttestationSigningKey, 
-    PolicyCertificatesModificationResult,
-    PolicyResult,
-    AttestationTokenValidationException
+    AttestationSigner,
+    AttestationToken,
+    AttestationPolicyResult,
+    AttestationPolicyCertificateResult,
+    AttestationTokenValidationException,
 )
 import base64
 from azure.core.tracing.decorator import distributed_trace
 from threading import Lock, Thread
+from ._common import pem_from_base64, validate_signing_keys, merge_validation_args
+from cryptography.x509 import load_pem_x509_certificate
 
 
 class AttestationAdministrationClient(object):
     """Provides administrative APIs for managing an instance of the Attestation Service.
 
-    :param instance_url: base url of the service
-    :type instance_url: str
+    The :class:`~AttestationAdministrationClient` object implements the policy
+    management and policy certificate management functions.
```

Comment:
Does this render correctly in the sphinx docs? It might need to be changed to `:class:\`~azure.security.attestation.AttestationAdministrationClient`

Reply:
The link seems to resolve correctly when I click on it. 

I've had to fix a bunch of related issues in parameters for ApiView, but apparently sphinx is ok with this one.